doc/lua_api.txt

```diff
@@ -1914,9 +1914,14 @@ and `minetest.auth_reload` call the authetification handler.
     * Returns `{name="ignore", ...}` for unloaded area
 * `minetest.get_node_or_nil(pos)`
     * Returns `nil` for unloaded area
-* `minetest.get_node_light(pos, timeofday)` returns a number between `0` and `15` or `nil`
+* `minetest.get_node_light(pos, timeofday)`
+    * Gets the light value at the given position. Note that if the given
+      position is a node, the light value "inside" the node will be returned.
+      So if you want to know the light level of a certain node, you need to
```

Comment:
There are non-air nodes that are transparent as well, like grass or glass. I think the criterion is if you set the drawtype to light.